src/NuGet.Core/NuGet.Build.Tasks.Console/NuGet.Build.Tasks.Console.csproj

```diff
@@ -57,6 +59,13 @@
     </AssemblyAttribute>
   </ItemGroup>
 
+  <Target Name="PackBuildOutputs">
+    <ItemGroup>
+      <TfmSpecificPackageFile Include="$(TargetPath)" PackagePath="contentFiles\any\$(TargetFramework)\" PackageCopyToOutput="true" />
+      <TfmSpecificPackageFile Include="$(ProjectRuntimeConfigFilePath)" Condition="'$(TargetFramework)' == '$(NetCoreTargetFramework)'" PackagePath="contentFiles\any\$(TargetFramework)\" PackageCopyToOutput="true" />
```

Comment:
Where are TargetPath and ProjectRuntimeConfigFilePath defined?

Reply:
They are global properties set during Build.

Reply:
Not really global properties but ones that come via the sdk and msbuild.